command/utils.go

```diff
@@ -0,0 +1,12 @@
+package command
+
+import (
+	"fmt"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+// PrintAppDetails prints some information about the app
+func PrintAppDetails(app *kingpin.Application) {
+	fmt.Println("help: ", app.Help)
```

Comment:
Use 'app.Usage' instead of 'app.Help' to print the application's usage information. The 'app.Help' field contains the long-form help message.